### DISCUSSION TITLE
Enhancing the UI of the feedback section

### DISCUSSION THREAD

**Discussion by shravanibhor** [chakra-ui/chakra-ui#8613] [Ideas]
### Description

The UI of the feedback section is not organized properly, isn't symmetrical, has empty spaces. My aim is to make it visually appealing.

### Problem Statement/Justification

![Screenshot (563)](https://github.com/chakra-ui/chakra-ui/assets/121655392/e597edaa-e207-4ced-a444-7470250d3536)
It clearly shows that the alignment of the boxes doesnt look good, either more boxes should be added or else they should be arranged such a way that they fit in with no empty spaces.

### Proposed Solution or API

My proposal was to make a horizontal infinite carousal, which will keep on moving horizontally, doesn't give space to any such unnecessary alignment issues.

**Comment by segunadebayo** [comment]
Closing. This is stale and we are not acting on it.
